vm_supervisor/storage.py

```diff
@@ -155,28 +161,93 @@ def create_ext4(path: Path, size_mib: int) -> bool:
     return True
 
 
+async def create_devmapper(volume: PersistentVolume | RootfsVolume, namespace: str) -> Path:
```

Comment:
This function is quite complex, some comments and documentation would be more than welcome.

Reply:
I just added some comments and link to documentation.